qa/rpc-tests/wallet_protectcoinbase.py

```diff
@@ -98,6 +98,14 @@ def run_test (self):
             else:
                 status = results[0]["status"]
                 errorString = results[0]["error"]["message"]
+
```

Comment:
Change Request: Verify that the result has a `method` field which equals `z_sendmany`.